### DISCUSSION TITLE
Reactタグの説明文(「React とは？」)にある「公式リファレンス」へのリンクが古い

### DISCUSSION THREAD

**Discussion by honey32** [increments/qiita-discussions#886] [要望・アイデア・不具合]
![image](https://github.com/user-attachments/assets/7fd9baf5-479c-40b5-885a-4042679c5b48)

新ドキュメントが正式版として https://react.dev/learn として公開されているのでリンク URL の変更をお願いします。

> These docs are old and won’t be updated. Go to [react.dev](https://react.dev/) for the new React docs.
> The new [Quick Start](https://react.dev/learn) teaches modern React and includes live examples.
> 
> 引用元: https://legacy.reactjs.org/docs/getting-started.html

# As Is

> 公式サイト: [React – A JavaScript library for building user interfaces](https://reactjs.org/)
> 公式リファレンス: [Getting Started – React](https://reactjs.org/docs/getting-started.html)
> Wikipedia: [React - Wikipedia](https://ja.wikipedia.org/wiki/React)

# To Be

> 公式サイト: [React – A JavaScript library for building user interfaces](https://reactjs.org/)
> 公式リファレンス: [Quick Start– React](https://react.dev/learn)
> Wikipedia: [React - Wikipedia](https://ja.wikipedia.org/wiki/React)

**Comment by tanaka-asami415** [comment]
@honey32 
ご報告ありがとうございます🙏変更いたしました！
https://qiita.com/tags/react

**Reply by honey32** [reply]
ありがとうございます！！
